docs/source/package_reference/trainer.mdx

```diff
@@ -1,69 +0,0 @@
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
--->
-
-# GaudiTrainer
-
-The [`GaudiTrainer`](https://huggingface.co/docs/optimum/habana/package_reference/trainer#optimum.habana.GaudiTrainer) class provides an extended API for the feature-complete [Transformers Trainer](https://huggingface.co/docs/transformers/main_classes/trainer). It is used in all the [example scripts](https://github.com/huggingface/optimum-habana/tree/main/examples).
-
-Before instantiating your [`GaudiTrainer`](https://huggingface.co/docs/optimum/habana/package_reference/trainer#optimum.habana.GaudiTrainer), create a [`GaudiTrainingArguments`] object to access all the points of customization during training.
-
-<Tip warning={true}>
-
-The [`GaudiTrainer`](https://huggingface.co/docs/optimum/habana/package_reference/trainer#optimum.habana.GaudiTrainer) class is optimized for 🤗 Transformers models running on Habana Gaudi.
-
-</Tip>
-
-Here is an example of how to customize [`GaudiTrainer`](https://huggingface.co/docs/optimum/habana/package_reference/trainer#optimum.habana.GaudiTrainer) to use a weighted loss (useful when you have an unbalanced training set):
-
-```python
-from torch import nn
-from optimum.habana import GaudiTrainer
-
-
-class CustomGaudiTrainer(GaudiTrainer):
-    def compute_loss(self, model, inputs, return_outputs=False):
-        labels = inputs.get("labels")
-        # forward pass
-        outputs = model(**inputs)
-        logits = outputs.get("logits")
-        # compute custom loss (suppose one has 3 labels with different weights)
-        loss_fct = nn.CrossEntropyLoss(weight=torch.tensor([1.0, 2.0, 3.0]))
-        loss = loss_fct(logits.view(-1, self.model.config.num_labels), labels.view(-1))
-        return (loss, outputs) if return_outputs else loss
-```
-
-Another way to customize the training loop behavior for the PyTorch [`GaudiTrainer`](https://huggingface.co/docs/optimum/habana/package_reference/trainer#optimum.habana.GaudiTrainer) is to use [callbacks](https://huggingface.co/docs/transformers/main_classes/callback) that can inspect the training loop state (for progress reporting, logging on TensorBoard or other ML platforms...) and take decisions (like early stopping).
-
-## GaudiTrainer
-
-[[autodoc]] transformers.trainer.GaudiTrainer
-    - all
-
-## GaudiSeq2SeqTrainer
-
-[[autodoc]] transformers.trainer_seq2seq.GaudiSeq2SeqTrainer
-    - evaluate
-    - predict
-
-## GaudiTrainingArguments
-
-[[autodoc]] transformers.training_args.GaudiTrainingArguments
-    - all
-
-## GaudiSeq2SeqTrainingArguments
-
-[[autodoc]] transformers.training_args_seq2seq.GaudiSeq2SeqTrainingArguments
-    - all
```

Comment:
why removed?

Reply:
Greg's vision of new doc org:
![image](https://github.com/user-attachments/assets/db8d47b7-31e6-431c-b2ce-5b39bebc059b)

Reply:
Added back after yours and regis' review